pkgs/development/libraries/science/math/m4rie/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchFromBitbucket
+, autoreconfHook
+, m4ri
+}:
+
+stdenv.mkDerivation rec {
+  version = "20150908";
+  name = "m4rie-${version}";
+
+  src = fetchFromBitbucket {
+    owner = "malb";
+    repo = "m4rie";
+    rev = "release-${version}";
+    sha256 = "0r8lv46qx5mkz5kp3ay2jnsp0mbhlqr5z2z220wdk73wdshcznss";
+  };
+
+  doCheck = true;
```

Comment:
Maybe
```
doInstallCheck = true;
```
would be a better option?

Reply:
There are no install tests.

Reply:
https://bitbucket.org/malb/m4rie/src/7c41c9a827e5f3831a90a7163e85c9de5892f3d6/INSTALL?at=master&fileviewer=file-view-default#INSTALL-70

Reply:
```
@nix { "action": "setPhase", "phase": "installCheckPhase" }
running install tests
installcheck flags: SHELL=/nix/store/xn5gv3lpfy91yvfy9b0i7klfcxh9xskz-bash-4.4-p19/bin/bash installcheck
Making installcheck in .
make[1]: Entering directory '/tmp/nix-build-m4rie-20150908.drv-0/source'
make[1]: Nothing to be done for 'installcheck-am'.
make[1]: Leaving directory '/tmp/nix-build-m4rie-20150908.drv-0/source'
Making installcheck in tests
make[1]: Entering directory '/tmp/nix-build-m4rie-20150908.drv-0/source/tests'
make[1]: Nothing to be done for 'installcheck'.
make[1]: Leaving directory '/tmp/nix-build-m4rie-20150908.drv-0/source/tests'
```

Reply:
Maybe that's because `make check` has already been run, but it shouldn't. Let's stay with `make check` then.